spec/features/ingredients/delete_ingredient_page_spec.rb

```diff
@@ -0,0 +1,14 @@
+feature 'user visits ingredients page' do
+  before do
+    FactoryBot.create(:ingredient, :egg)
+    visit('/ingredients')
+    click_link('Delete')
```

Comment:
The test should be more robust:

1. Create two ingredients.
2. Click `Delete` for the second one.
3. Verify that first one still exists and second one is no longer there.